AGENT_INTEGRATIONS.md

```diff
@@ -906,7 +1037,7 @@
 * datadog-gearmand: 1.3.0
 * datadog-gitlab-runner: 2.3.0
 * datadog-gitlab: 2.3.0
-* datadog-go-expvar: 1.2.0
+* datadog-go-expvar: 1.3.0
```

Comment:
This is correct: `go_expvar` was in version `1.3.0` as of the `6.12.2` tag, and it was listed as `1.3.0`in `requirements-agent-release.txt`.

Reply:
Agree. Just to confirm:

```
$ docker run datadog/agent:6.12.0 /opt/datadog-agent/embedded/bin/pip freeze | grep datadog-go-expvar
datadog-go-expvar==1.3.0
```